src/main/resources/db/migration/all/20240513120304__add_application_prison_release_types_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE application_prison_release_types (
```

Comment:
I think this should maybe be called 'temporary_accommodation_application_prison_release_types' as it's only applicable to cas3 application?

Reply:
Changed. I named it as application_prison_release_types if another CAS need to use it in the future

Reply:
I was considering that but noticed we had a simple table for CAS1 that was prefixed with approved_premises_applicaitons. we can always rename (and move relationship to the common application entity in code) if needed by other CASs